app/javascript/components/dashboard-form/dashboard-form.schema.js

```diff
@@ -0,0 +1,50 @@
+import { componentTypes, validatorTypes } from '@data-driven-forms/react-form-renderer';
+
+function createSchema(maxNameLen, maxDescLen, readOnly) {
+  const fields = [
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'name',
+      label: __('Name'),
+      maxLength: maxNameLen,
+      validate: [
+        {
+          type: validatorTypes.REQUIRED,
+          message: __('Required')
+        },
+        {
+          type: validatorTypes.MAX_LENGTH,
+          message: _('Maximum allowed length is 20 characters.')
+        },
+        {
+          type: validatorTypes.PATTERN_VALIDATOR,
+          pattern: '(.)*(\\|)',
```

Comment:
i dont think this regex will work.

It might be easier to just pass function instead of the validator object in this case
```javascript
const includeValidator = (value, string) => value.includes(string)
```

And use it like this

```javascript
validate: [{
  ...
}, 
value => includeValidator(value, '\'),
{...}
]
```